packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -162,10 +172,10 @@ type Props = {
    */
   tokenSelector?: ReactNode
 
-  /**
-   * Maximum balance configuration for the input.
-   * {@link MaxBalanceProps}
-   */
+  /** Optional wallet balance configuration. */
+  walletBalance?: WalletBalanceProps
```

Comment:
Out of scope for this PR, but I just wanted to mention that this smells like too much coupling.
Whenever we need to pass properties to another component deeper in the tree, we could try using `children` or some other `ReactNode` prop.